vmdb/spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -5,7 +5,7 @@
   before(:each) do
     set_user_privileges
     @zone = FactoryGirl.create(:zone, :name => 'zone1')
-    @provider = ProviderForeman.create(:name => "test", :url => "10.8.96.102", :verify_ssl => nil, :zone => @zone)
+    @provider = ProviderForeman.create(:name => "test", :url => "10.8.96.102", :verify_ssl => '1', :zone => @zone)
```

Comment:
Should this be an integer 1, not a string 1?  Why not just use the real SSL constant for clarify and to avoid "magic numbers"?